dataflows/processors/dumpers/file_formats.py

```diff
@@ -203,3 +203,36 @@ def write_transformed_row(self, transformed_row):
 
     def finalize_file(self):
         self.writer.write(']')
+
+
+class GeoJSONFormat(JSONFormat):
+    def __init__(self, file, schema, **options):
+        writer = file
+        writer.write('{"type": "FeatureCollection","features":')
+        writer.__first = True
```

Comment:
not necessary as you're calling `JSONFormat` constructor

Reply:
Yes, but how do I initialize the GeoJSON file with  ```{"type": "FeatureCollection","features":```?

Reply:
Okay I see what you mean.